app/security/WebAuthnCredentialRepository.scala

```diff
@@ -0,0 +1,122 @@
+package security
+
+import models.user.{MultiUserDAO, WebAuthnCredential, WebAuthnCredentialDAO}
+import com.yubico.webauthn._
+import com.yubico.webauthn.data._
+import com.scalableminds.util.accesscontext.GlobalAccessContext
+import com.scalableminds.util.objectid.ObjectId
+import com.typesafe.scalalogging.LazyLogging
+import net.liftweb.common.{Empty, Failure, Full}
+
+import java.util.Optional
+import javax.inject.Inject
+import scala.annotation.nowarn
+import scala.jdk.CollectionConverters._
+
+object WebAuthnCredentialRepository {
```

Comment:
Please remove unnecessary brackets and semicolons in this file as well :see_no_evil: